templates/IAM/github-oidc-provider-access.njk

```diff
@@ -0,0 +1,115 @@
+{# Allow Github to access AWS without giving github AWS credentials               #}
+{# Must pass in the following templating parameters                               #}
+{#  Parameters:                                                                   #}
```

Comment:
we should probably add some comments similar to https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/650-identity-providers/github-oidc-provider-access.njk#L1-L11